### PR TITLE
Updated the Dev branch

### PR DESCRIPTION
**Pull request by sushpawar001**
Added docstrings according to the PEP. Optimized a few functions.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -35,23 +41,38 @@
...
 end_seconds = 0
 previous_day = args.day
-# ************************************************* GLOBAL FUNCTIONS ************************************************* #
 
 
+# ************************************************* GLOBAL FUNCTIONS ************************************************* #
 def edit_flair() -> bool:
-    """Deletes and sets user flair, then updates the wiki pages.
+    """
+    Deletes and sets user flair, then updates the wiki pages.
```
Comment (anthonyLuttrell) [inline main.py:75]: Please use the Google style guide as a reference for docstrings

https://google.github.io/styleguide/pyguide.html#s3.8-comments-and-docstrings

> A docstring should be organized as a summary line (one physical line not exceeding 80 characters) terminated by a period, question mark, or exclamation point. When writing more (encouraged), this must be followed by a blank line, followed by the rest of the docstring starting at the same cursor position as the first quote of the first line.

File: main.py [hunk 2]
```diff
@@ -257,86 +369,97 @@ def sys_exit():
 
 
 # **************************************************** MAIN LOOP ***************************************************** #
-
-try:
+if __name__ == "__main__":
```
Comment (anthonyLuttrell) [inline main.py:383]: Remove this for now, the dev_oop branch will include a main function.

**Review by anthonyLuttrell — CHANGES_REQUESTED** [review]
Please fix the docstring syntax and remove the if main check and I will try to get this merged into the master branch soon.

**Comment by sushpawar001** [comment]
Please check it out. Made the requested changes.

**Pull request merged**